collection-renderer-lib/src/cmr/collection_renderer/services/collection_renderer.clj

```diff
@@ -24,11 +26,13 @@
   "The main ERB used to generate the Collection HTML."
   (io/resource "collection_preview/collection_preview.erb"))
 
-(def preview-gem-umm-schema-version
-  "Temporay def for the UMM schema version that is supported by the CMR preview gem.
-   This will be removed once CMR preview gem is fixed to include the UMM schema version
-   it supports in the gem. (MMT-921)"
-  "1.6")
+(def preview-gem-umm-version-config-file
+  "Defines the path to the UMM schema version config file within the preview gem."
+  "gems/cmr_metadata_preview-0.0.1/.umm-version")
```

Comment:
Talked to @abbottry, filed MMT-927 for providing versioning in preview gem. I will put this on hold until we resolve the versioning issue.